clang/lib/CodeGen/CodeGenPGO.h

```diff
@@ -112,6 +112,7 @@ class CodeGenPGO {
 public:
   std::pair<bool, bool> getIsCounterPair(const Stmt *S) const;
```

Comment:
I think this function could use some documentation, or a better name.

This can happen in a later commit.